frontend/packages/console-shared/src/components/dashboard/inventory-card/InventoryItem.tsx

```diff
@@ -203,13 +204,15 @@ export const ResourceInventoryItem: React.FC<ResourceInventoryItemProps> = ({
   ExpandedComponent,
   basePath,
 }) => {
-  const TitleComponent = React.useCallback(
+  let Title: React.ComponentType = React.useCallback(
     (props) => (
       <ResourceTitleComponent kind={kind} namespace={namespace} basePath={basePath} {...props} />
     ),
     [kind, namespace, basePath],
   );
 
+  if (TitleComponent) Title = TitleComponent;
```

Comment:
```suggestion
let Title: React.ComponentType;
  if (TitleComponent) {
    Title = TitleComponent;
  } else {
    Title = React.useCallback(
      (props) => (
        <ResourceTitleComponent kind={kind} namespace={namespace} basePath={basePath} {...props} />
      ),
      [kind, namespace, basePath],
    );
  }
```

Reply:
@a2batic that would call hook conditionally which is not allowed.

I think the current solution is fine.

Reply:
ack, Thanks @rawagner.